similarity/src/main/java/edu/illinois/cs/cogcomp/nesim/io/StandardInputOutput.java

```diff
@@ -0,0 +1,207 @@
+/**
```

Comment:
Don't change this class for the current PR, but please create an issue: this functionality should be replace with core-utilities classes.